hw/ip/sram_ctrl/doc/checklist.md

```diff
@@ -222,10 +222,10 @@ Review        | [V3_CHECKLIST_SCOPED][]                 | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Not Started |
-Tests         | [FPV_SEC_CM_VERIFIED][]                 | Not Started |
-Tests         | [SIM_SEC_CM_VERIFIED][]                 | Not Started |
-Review        | [SEC_CM_DV_REVIEWED][]                  | Not Started |
+Documentation | [SEC_CM_TESTPLAN_COMPLETED][]           | Done        |
+Tests         | [FPV_SEC_CM_VERIFIED][]                 | Done        | Sub-module prim_count is proven in FPV
```

Comment:
@cindychip 
We are also proving the IP itself also in an FPV testbench, right? To prove the error from `prim_counts` always reaches the alert output unhindered.

Reply:
yes, Cindy helped to add it and it's passing now according to https://reports.opentitan.org/hw/top_earlgrey/formal/summary.html.